### ISSUE TITLE
Missing description in TNDS timetable information

### ISSUE THREAD

**Issue by jw35** [SmartCambridge/tfc_web#268]
```
[2019-01-02 03:30:30,997] [transport.management.commands.update_bus_info] [ERROR] - Error while trying to process file bed_81-1D-_-y08-5.xml, exception was 'Description'
Traceback (most recent call last):
  File "/home/tfc_prod/tfc_web/tfc_web/transport/management/commands/update_bus_info.py", line 82, in handle
    description=service['Description'],
KeyError: 'Description'
```

**Comment by jw35** [comment]
bed_46-26-_-y08-1.xml has joined in throwing this error

**Comment by jw35** [comment]
Fixed in 7b5dc990
